tests/validation_tool/icmp.rflx

```diff
@@ -0,0 +1,118 @@
+package ICMP is
+
```

Comment:
I might be wrong but didn't we refrain from using IPv4 because Linux does not check the header checksum anyway?

Reply:
I don't remember that this conclusion was drawn. The meeting minutes of 2021-06-10 explicitly say that the tool "[s]hould be tested with IPv4 and ICMP". Even if it would be the case that Linux does not check the header checksum: RecordFlux is not limited to Linux. A full specification and implementation of IPv4 needs a definition of the checksum.

Reply:
Added checksum aspect in 8c0da0c3, checksum function in 37d96a4c